.goreleaser.yml

```diff
@@ -8,25 +8,36 @@ before:
     # - go generate ./...
 builds:
   - binary: aws-resource-counter
-    id: crc-others
+    id: arc-others
     goos:
       - linux
   - binary: aws-resource-counter
-    id: crc-macos
+    id: arc-macos
     goos:
       - darwin
     goarch:
       - amd64
       - arm64
     hooks:
-      post: gon gon.hcl
+      post:
+        - |
+          sh -c '
+          cat > /tmp/arc-{{ .Arch }}-gon.hcl <<EOF
+          source = ["./dist/arc-macos_{{ .Target }}/{{ .Name }}"]
+          bundle_id = "com.expel.aws-resource-counter"
+          sign {
+            application_identity = "71c685df24be2279027ca972134afd5c715ed841"
+          }
+          EOF
+          '
```

Comment:
just to make sure i understand this right, this is just making the two gon.hcl files? so instead of having the two hcl files committed in the repository, it's just being created through the build?

Reply:
yep, and if we end up adding any other targets in the future, this will take care of it without needing an extra config file